composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/twig-bridge": "^2.3.5 || ^3.0",
         "symfony/validator": "^2.3 || ^3.0",
         "twig/extensions": "^1.0",
-        "twig/twig": "^1.26"
+        "twig/twig": "^1.28"
```

Comment:
Actually, it does not seem to be a BC break http://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api